content/posts/newsletter-013/index.md

```diff
@@ -66,6 +66,16 @@ If needed, a section can be split into subsections with a "------" delimiter.
 
 ## Library & Tooling Updates
 
+### [blitz-path](https://github.com/BezPowell/blitz-path)
```

Comment:
```suggestion
### [blitz-path](https://github.com/BezPowell/blitz-path)

```

Reply:
Linter is now complaining that there needs to be an extra linebreak here (why didn't it say that the first time 🤔)